### DISCUSSION TITLE
Will Azure DevOps Repos support be included in the roadmap?

### DISCUSSION THREAD

**Discussion by sheldonhull** [community/community#4151] [Codespaces] [answered]
I love working with codespaces, but with my primary work repos being in Azure DevOps Repos, I'm unable to enjoy the benefit of codespaces.

Visual Studio Codespaces was deprecated so now it's just GitHub codespaces.

I've found a few references to plans to enable, but nothing concrete.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @sheldonhull 👋 Apologies for the delay here. 
Currently, we don't have immediate plans to add support for ADO repos, however we have a workaround that enables using Codespaces with ADO repos: https://github.com/codespaces-contrib/azdo-in-codespaces

Hope this helps!
